tests/postgresql/test-postgresql.cpp

```diff
@@ -158,23 +159,59 @@ TEST_CASE("PostgreSQL blob", "[postgresql][blob]")
 
         blob_table_creator tableCreator(sql);
 
-        char buf[] = "abcdefghijklmnopqrstuvwxyz";
-
-        sql << "insert into soci_test(id, img) values(7, lo_creat(-1))";
```

Comment:
Here again, I'm not sure if this was changed because this doesn't work any longer or some other reason?

Reply:
It's not needed anymore